pages/orgs/[orgId]/settings.tsx

```diff
@@ -547,103 +552,110 @@ export default function EditOrgPage() {
             {users?.length == 0 && <p className="mt-4">no data to show</p>}
             {userRole === "ADMIN" && (
               <>
-                <Heading className="text-center mt-16 mb-8">
-                  Your subscription
-                </Heading>
-                {!loading && subs?.length != 0 && (
-                  <div>
-                    <Table
-                      className="mt-8"
-                      sx={{ minWidth: 650, maxWidth: 1000 }}
-                      aria-label="simple table"
-                    >
-                      <Thead>
-                        <Tr>
-                          <Th>
-                            <strong>Org Id</strong>
-                          </Th>
-                          <Th>
-                            <strong>Organisation</strong>
-                          </Th>
-                          <Th>
-                            <strong>Subscription</strong>
-                          </Th>
-                          <Th></Th>
-                        </Tr>
-                      </Thead>
-                      <Tbody>
-                        {subs?.map((sub, index) => {
-                          return (
-                            <Tr key={index}>
-                              <Td>{sub.org.id}</Td>
-                              <Td>{sub.org.name}</Td>
-                              <Td>
-                                <Tag
-                                  size={"md"}
-                                  key={index}
-                                  borderRadius="full"
-                                  variant="solid"
-                                  colorScheme={getColorLabel(sub?.subName)}
-                                >
-                                  {translateSubName(sub.subName)}
-                                </Tag>
-                              </Td>
-                            </Tr>
-                          );
-                        })}
-                      </Tbody>
-                    </Table>
-
-                    <Center>
-                      <Button
-                        rightIcon={<FiExternalLink />}
-                        colorScheme="blue"
-                        variant="solid"
-                        className="mt-4"
-                        onClick={sendCreateCustomerPortalSession}
-                      >
-                        manage subscription
-                      </Button>
-                    </Center>
-                    {loading && (
+                {parseBooleanEnvValue(
+                  window.__env.NEXT_PUBLIC_STRIPE_ENABLED
+                ) && (
```

Comment:
Why do you access the window.__env directly instead of using loadConfig which uses it to configure the configuration values?